releasenotes/notes/feat-change-default-riot-path-with-a-env-var-12d2c59e74af9599.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add `RIOT_ENV_BASE_PATH` environment variable to change the default path of `.riot` directory instead of current
```

```suggestion
    Add ``RIOT_ENV_BASE_PATH`` environment variable to change the default path of ``.riot`` directory instead of current
```
I think we need to double-backtick-quote this? I can never remember 😢 @brettlangdon @Kyle-Verhoog can you confirm?

---

Yeah Gab you're correct, we need double backticks for envars.